user-console/source/org/pentaho/mantle/public/browser/messages_fr.properties

```diff
@@ -2,7 +2,7 @@
 # *******************************************************************************
 # Pentaho
 #
-# Copyright (C) 2002-2013 by Pentaho : http://www.pentaho.com
+# Copyright (C) 2002-2013 by Pentaho\u0020: http://www.pentaho.com
```

Comment:
I can tell someone did a search and replace on " :" with "\u0020:" ... right @kcruzada?